src/lib/statistics/base_column_statistics.cpp

```diff
@@ -22,4 +22,10 @@ std::shared_ptr<BaseColumnStatistics> BaseColumnStatistics::without_null_values(
   return clone;
 }
 
+std::shared_ptr<BaseColumnStatistics> BaseColumnStatistics::only_null_values() const {
```

Comment:
I'm not super happy about this but not sure what to do.
If there is a statistics object that says there are only null values then distinct count and min/max should not be set to inconsistent values.
This seems like a larger problem, though, and not part of this PR.
I wanted to mention it and hear what you think about it.
My proposal would be to introduce an additional type of column statistics that represents a column full of null values.

Reply:
The BaseColumnStatistics are such a zombie that this method feels almost clean to me. I'd leave it as it is, added this function mainly for consistency with `without_null_values()`